pom.xml

```diff
@@ -99,6 +99,11 @@
       <artifactId>video-js</artifactId>
       <version>4.11.4</version>
 	</dependency>
+  <dependency>
```

Comment:
It's a details but it would be nicer to indent this <dependency> element properly.